styles/globals.scss

```
@@ -118,3 +118,43 @@ body {
 .cursor-pointer {
     cursor: pointer;
 }
+#notification {
+    animation: slide-in 1s ease-in-out normal;
+    overflow: hidden;
+}
+
+@keyframes slide-in {
+    0% {
+        opacity: 0;
+        transform: translateY(-100%);
+    }
+    100% {
+        opacity: 1;
+        transform: translateY(0);
+    }
+}
+
+#notification::after {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 0.2rem;
+    position: absolute;
+    bottom: 0;
+    margin-bottom:-0.05rem;
+    opacity: 0.2;
+    left: 0;
+    background-color: rgba(0, 0, 0, 0.253);
```

I think you carely can see it because you left `opacity:0.2;` and `background-color: rgba(0, 0, 0, 0.253);`  
```suggestion
    background-color: rgb(0 0 0);
```
with this or a lighter opacity it looks good!

---

You are right, I missed the rgba :joy_cat: thanks.